plugin/sampling/strategystore/adaptive/options_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
remove this header and run `make fmt`

Reply:
Hello @yurishkuro , I removed the header as  instructed  but the `make fmt` command seems to be revealing  errors on files  I did not edit. Please is this the expected behavior of the `make fmt` command ?

![image](https://github.com/jaegertracing/jaeger/assets/156124227/5a1ed63f-fd3e-452d-a569-d51dc3895815)

Reply:
seems your toolchain may not be up to date: Go 1.21 and `make install-tools`

Reply:
Thank you. The problem was , I had two different go versions installed.